content/_index.fr.md

```diff
@@ -0,0 +1,9 @@
+{
+    "title": "demandetesdonnees.fr – Utilise ton droit de confidentialité.",
+    "_title": "demandetesdonnees.fr – Utilise ton droit de confidentialité.",
```

Comment:
It would be more like:
```diff
-"title": "demandetesdonnees.fr – Utilise ton droit de confidentialité.",
-"_title": "demandetesdonnees.fr – Utilise ton droit de confidentialité.",
+"title": "demandetesdonnees.fr – Utilise ton droit à la vie privée.",
+"_title": "demandetesdonnees.fr – Utilise ton droit à la vie privée.",
```